### ISSUE TITLE
README refers to nonexistent directoriy

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#165]
```
README refers to docs/authentication.txt, file is actually in 
doc/authentication.txt.
```

Original issue reported on code.google.com by `michael....@dianomi.com` on 25 Oct 2012 at 5:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `niels.he...@gmail.com` on 17 Nov 2012 at 4:50
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Cheers! This will be fixed in the next release
```

Original comment by `niels.he...@gmail.com` on 17 Nov 2012 at 4:50
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.10b (currently in SVN). Thanks for reporting !
```

Original comment by `niels.he...@gmail.com` on 23 Nov 2012 at 8:52
- Changed state: **Fixed**
